src/ui/public/chrome/directives/global_nav/global_nav.js

```diff
@@ -50,6 +50,9 @@ module.directive('globalNav', (globalNavState, chrome) => {
         globalNavState.setOpen(!globalNavState.isOpen());
       };
 
+      scope.isHomeActive = () => {
+        return true;
```

Comment:
@nreese Any suggestions on the right way to figure out if we're on the home page?

Reply:
you could inspect the `window.location` and check the fragment value and see if its `home`

Reply:
Works like a charm, thanks.